### PR TITLE
Add Disclaimer

### PR DESCRIPTION
**Pull request by Sneh1999**
**What changed? Why?**
Add a disclaimer based on Rose's design
image

**Notes to reviewers**

**How has it been tested?**

### CODE REVIEW

File: apps/build-onchain-apps/app/buy-me-coffee/index.tsx [hunk 1]
```diff
@@ -21,7 +22,9 @@ export default function BuyMeCoffeePage() {
   return (
     <>
       <Header />
-      <main className="container mx-auto flex flex-col">
+      <main className="container mx-auto flex flex-col px-8">
+        <Disclaimer />
```
Comment (Zizzamia) [inline apps/build-onchain-apps/app/buy-me-coffee/index.tsx:26]: Nice! Let's add this in the other two mint pages.

File: apps/build-onchain-apps/src/components/disclaimer/Disclaimer.tsx [hunk 2]
```diff
@@ -0,0 +1,32 @@
+import Image from 'next/image';
+
+export default function Disclaimer() {
+  return (
+    <div className="mt-28 flex w-full flex-col items-center justify-between rounded-xl bg-[#141519] p-4 text-base text-white md:mt-12 md:flex-row lg:mt-24">
```
Comment (Zizzamia) [inline apps/build-onchain-apps/src/components/disclaimer/Disclaimer.tsx:5]: A couple notes:
1. add the bg color in the extra colors in the tailwind configuration https://github.com/coinbase/build-onchain-apps/blob/main/apps/build-onchain-apps/tailwind.config.ts#L32 as makes them easier to reuse and understand
2. you can split this long line with `clx` as you can see here https://github.com/coinbase/build-onchain-apps/blob/main/apps/build-onchain-apps/src/components/header/AccountDropdown.tsx#L29 as that will make the css more readiable.

**Review by Zizzamia — APPROVED** [review]
Nice, feel free to make adjust the code with the notes I wrote either in this PR or in the next one.

File: apps/build-onchain-apps/src/components/disclaimer/Disclaimer.tsx (hunk 2 above)
Comment (Sneh1999) [reply apps/build-onchain-apps/src/components/disclaimer/Disclaimer.tsx]: Makes sense! Updated this

**Comment by Sneh1999** [comment]
> Nice, feel free to make adjust the code with the notes I wrote either in this PR or in the next one.

@Zizzamia  updated based on comments, feel free to review again

**Review by Zizzamia — APPROVED** [review]

**Pull request merged**